microsoft-edge/extensions-chromium/developer-guide/sidebar.md

```diff
@@ -0,0 +1,625 @@
+    });
+  } else {
+    // Disables the sidebar when at other sites
+    await chrome.sidePanel.setOptions({
+      tabId,
+      enabled: false
+    });
+  }
+});
+```
+
+When a user switches to a tab or site for which the sidebar is not enabled, the sidebar is hidden.
+
+For a complete example, see the [Site-specific side panel example](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/functional-samples/cookbook.sidepanel-site-specific) sample.
+
+
+<!-- ------------------------------ -->
+#### Enable the extension's shortcut icon to open the sidebar
+
+<!-- compare https://developer.chrome.com/docs/extensions/reference/sidePanel/#open-action-icon -->
+
+To allow users to open the sidebar by clicking the action toolbar icon, use [sidePanel.setPanelBehavior()](https://developer.chrome.com/docs/extensions/reference/sidePanel/#method-setPanelBehavior).  First, declare the `"action"` key in the manifest:
+
+`manifest.json`:
+
+```json
+{
+  "name": "My sidebar extension",
+  ...
+   "action": {
+    "default_title": "Click to open sidebar"
+  },
+  ...
+}
+```
+
+Then, add the following code to the code listing that's in [Enable a sidebar for a specific site only](#enable-a-sidebar-for-a-specific-site-only) above:
+
+`service-worker.js`:
+
+```js
+const BING_ORIGIN = 'https://www.bing.com';
+
+// Allow users to open the sidebar by clicking on the action toolbar icon
+chrome.sidePanel
+  .setPanelBehavior({ openPanelOnActionClick: true })
+  .catch((error) => console.error(error));
+...
+```
+
+
+<!-- ------------------------------ -->
+#### Switch to a different sidebar
+
+<!-- compare https://developer.chrome.com/docs/extensions/reference/sidePanel/#multi-panels -->
+
+An extension can use [sidepanel.getOptions()](https://developer.chrome.com/docs/extensions/reference/sidePanel/#method-getOptions) to retrieve the current sidebar, and then enable a different sidebar for a specific tab.
+
+This example sets a sidebar containing a welcome message on [runtime.onInstalled()](https://developer.chrome.com/docs/extensions/reference/runtime/#event-onInstalled).  When the user navigates to a different tab, the sidebar is replaced with the browser-level sidebar.
+
+`service-worker.js`:
+
+```js
+const welcomePage = 'sidebar/welcome-sp.html';
+const mainPage = 'sidebar/main-sp.html';
+
+chrome.runtime.onInstalled.addListener(() => {
+  chrome.sidePanel.setOptions({ path: welcomePage });
+});
+
+chrome.tabs.onActivated.addListener(async ({ tabId }) => {
+  const { path } = await chrome.sidePanel.getOptions({ tabId });
+  if (path === welcomePage) {
+    chrome.sidePanel.setOptions({ path: mainPage });
+  }
+});
+```
+
+
+<!-- ====================================================================== -->
+## The sidebar user experience (UX)
+
+<!-- compare https://developer.chrome.com/docs/extensions/reference/sidePanel/#user-experience -->
+
+Extensions in the Microsoft Edge sidebar have these user experience (UX) features.
```

Comment:
"Extensions in the Microsoft Edge sidebar" seems incorrect. Maybe something like "Extensions using the sidebar in Microsoft Edge" would be more appropriate?